macros/plot_vfat_and_channel_Scurve.py

```diff
@@ -1,4 +1,4 @@
-from plot_scurve import *
+from macros.plot_scurve import *
```

Comment:
This script seems to only add options on top of the `plot_scurve` script.  If that's all there is, suggest to merge `plot_vfat_and_channel_Scurve.py` and `plot_scurve.py`, making it similar to the other plotting macros, and just make it standalone (though my quick glance at the `macros` may have missed something in how this is used).